src/servers/src/http/event.rs

```diff
@@ -602,7 +606,8 @@ fn extract_pipeline_value_by_content_type(
                     }
                 };
 
-                if let Ok(v) = serde_json::from_str(&line) {
+                if let Ok(v) = unsafe { simd_json::to_owned_value(line.as_mut_vec()) } {
```

Comment:
Can the safety be hold? [`as_mut_vec`](https://doc.rust-lang.org/std/string/struct.String.html#method.as_mut_vec) safety requires that bytes are still valid UTF-8 after changed. Does the `to_owned_value` guarantee that?

Reply:
Change `as_mut_vec` to `as_bytes_mut`, for it doesn't require the unstable feature.
From the description, `to_owned_value` only de-escapes the string. It should work on a character level like any other JSON deserializer.

Reply:
safety should be commented